make/modules/jdk.internal.le/Lib.gmk

```diff
@@ -27,14 +27,15 @@ include LibCommon.gmk
 
 ################################################################################
 
-ifeq ($(call isTargetOs, windows), true)
+ifeq ($(call isTargetOs, linux macosx windows), true)
 
   $(eval $(call SetupJdkLibrary, BUILD_LIBLE, \
       NAME := le, \
       OPTIMIZATION := LOW, \
-      CFLAGS := $(CFLAGS_JDKLIB), \
+      CFLAGS := $(CXXFLAGS_JDKLIB), \
       LDFLAGS := $(LDFLAGS_JDKLIB), \
-      LIBS := $(JDKLIB_LIBS) user32.lib, \
+      LIBS_windows := $(JDKLIB_LIBS) user32.lib, \
+      LIBS_unix := $(JDKLIB_LIBS) -lstdc++, \
```

Comment:
If this is a C++ library, I think it's better to specify `TOOLCHAIN := TOOLCHAIN_LINK_CXX` than adding `-lstdc++`.

Reply:
We may also need `$(LIBCXX)` to force static linking when that is enabled for the build.